### PR TITLE
Tunnelblick

### LINKED ISSUE #409: streisand-mirror : Mirror Tunnelblick for OS X FAILED
TASK [streisand-mirror : Mirror Tunnelblick for OS X] **************************
fatal: [45.33.54.248]: FAILED! => {"changed": false, "dest": "/var/www/streisand/mirror/openvpn", "failed": true, "gid": 33, "group": "www-data", "mode": "0755", "msg": "Request failed", "owner": "www-data", "response": "HTTP Error 404: Not Found", "size": 4096, "state": "directory", "status_code": 404, "uid": 33, "url": "https://tunnelblick.net/release/Tunnelblick_3.6.6_build_4582.dmg"}

### PR DESCRIPTION
**Pull request by alimakki**
Updated tunnelbick to latest version, resolved  #409

### CODE REVIEW

File: playbooks/roles/streisand-mirror/vars/openvpn.yml [hunk 1]
```diff
@@ -40,9 +40,9 @@ openvpn_download_urls:
   - "{{ openvpn_windows64_installer_sig_url }}"
 
 # OS X
-tunnelblick_version: "3.6.6"
-tunnelblick_build: "4582"
```
Comment (dasrecht) [inline playbooks/roles/streisand-mirror/vars/openvpn.yml]: The build is 4606 by now

File: playbooks/roles/streisand-mirror/vars/openvpn.yml [hunk 2]
```diff
@@ -40,9 +40,9 @@ openvpn_download_urls:
...
+tunnelblick_version: "3.6.7a"
```
Comment (dasrecht) [inline playbooks/roles/streisand-mirror/vars/openvpn.yml]: This is 3.6.7.c by now

**Comment by alimakki** [comment]
Updated again!

**Comment by alimakki** [comment]
Updated!

**Comment by dasrecht** [comment]
Tested and looks good to me 👍

**Comment by alimakki** [comment]
Excellent glad I could help :)

**Comment by jlund** [comment]
Thanks!

**Pull request merged**